src/data/sfsmaster.cfg.in

```diff
@@ -241,6 +241,11 @@
 ## (Default: 3600)
 # METADATA_DUMP_PERIOD_SECONDS = 3600
 
+## Periodical empty reserved files interval in milliseconds. If set to 0, the reserved files 
```

Comment:
Consider: Interval for periodically cleaning of reserved files, in milliseconds.

Reply:
Done